coriolis/tests/api/v1/views/test_region_view.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from coriolis.api.v1.views import region_view
+from coriolis.api.v1.views import utils as view_utils
+from coriolis.tests import test_base
+
+
+class RegionViewTestCase(test_base.CoriolisApiViewsTestCase):
+    """Test suite for the Coriolis api v1 views."""
+
+    def setUp(self):
+        super(RegionViewTestCase, self).setUp()
+        self._format_fun = region_view._format_region
+
+    @mock.patch.object(view_utils, 'format_opt')
+    def test_format_region(self, mock_format_opt):
+            mock_format_opt.return_value = {
+                "mapped_endpoints": [{'id': 'mock_id1'}, {'id': 'mock_id2'}],
```

Comment:
Please change `mock_id1` and `mock_id2` to `endpoint_1` / `service_1` and `endpoint_2` / `service_2` respectively, to avoid a chance these would get mixed up.